internal/provisioner/public_nginx.go

```diff
@@ -73,11 +81,17 @@ func (n *publicNginx) Destroy() error {
 }
 
 func (n *publicNginx) NewHelmDeployment() *helmDeployment {
+	awsACMCert, err := n.awsClient.GetCertificateSummaryByTag(aws.DefaultCloudDNSTagKey, aws.DefaultPublicCloudDNSTagValue, n.logger)
```

Comment:
@cpanato how are we planning to create this Cert? I guess with Terraform when we deploy the provisioner in C&C?

Reply:
the certificate is already created
for test is *.test.mattermost.cloud and so on.
we are using the same we used to use before

Reply:
Athh the one we were using before. We need to make sure we have this everywhere. Cool thanks :)